test/test_groth16_verifier.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
```

Comment:
move the command `generate_verifiler` to snark_verifier.sh, and remove this file.